package/src/kustoMode.ts

```diff
@@ -21,16 +19,19 @@ let workerPromise: Promise<WorkerAccessor> = new Promise((resolve, reject) => {
  * Called when Kusto language is first needed (a model has the language set)
  * @param defaults
  */
-export function setupMode(defaults: LanguageServiceDefaultsImpl, monacoInstance: typeof monaco): WorkerAccessor {
+export function setupMode(
+    defaults: LanguageServiceDefaultsImpl,
+    monacoInstance: typeof globalThis.monaco
```

Comment:
```suggestion
    monacoInstance: monaco
```

I'm not too familiar with the types here but we can't reference the `monaco` type that's from the import statement on line 1 (e.g. `import * as monaco from 'monaco-editor'`?

Reply:
Disregard.